reportportal_client/helpers.py

```diff
@@ -106,3 +106,27 @@ def get_package_version(package_name):
     except DistributionNotFound:
         package_version = 'not found'
     return package_version
+
+
+def verify_value_length(attributes):
+    """Verify length of the attribute value.
+
+    The length of the attribute value should have size from '1' to '128'.
+    Otherwise HTTP response will return an error.
+    Example of the input list:
+    [{'key': 'tag_name', 'value': 'tag_value1'}, {'value': 'tag_value2'}]
+    :param attributes: List of attributes(tags)
+    :return:           List of attributes with corrected value length
+    """
+    if attributes is not None:
+        for pair in attributes:
+            if not isinstance(pair, dict):
```

Comment:
Is it a valid case? Just being curious, because it is not considered in unittests.

Reply:
Will add unit tests. Yeah, as it is a public function, we have to take into account all the scenarious.